mailpoet/tasks/phpstan/phpstan.neon

```diff
@@ -42,8 +42,6 @@ parameters:
     - '/Parameter #1 \$cssOrXPath of method AcceptanceTester::moveMouseOver\(\) expects string\|null, array<string, string> given./'
     - '/Function expect invoked with 1 parameter, 0 required\./'
     - '/Call to method getName\(\) on an unknown class _generated\\([a-zA-Z])*Cookie/' # codeception generate incorrect return type in ../../tests/_support/_generated
-    - '/Call to static method container\(\) on an unknown class/'
```

Comment:
It is not immediately obvious to me which changes in this commit prompted the need to update the phpstan.neon file to remove a few of the ignore errors. Could you please elaborate?

Reply:
When I changed the class in the integration, I got new warnings from the PHPStan complaining that the `Automattic\WooCommerce\StoreApi\Schemas\V1\CheckoutSchema` doesn't exist.
So I could add a new ignore rule to the phpstan.neon, but I checked the WooCommerce Stubs and saw that the newer version contains the class. 
After I updated the Woo Stubs, some previously thrown errors were not thrown any more because after updating stubs PHPStan knows the new stuff in Woo.